packages/Babylonian-Core.package/BPExampleToTest.class/instance/buildMethodCall.st

```diff
@@ -0,0 +1,7 @@
+building
+buildMethodCall
+	"Builds the method body as string. example: '(Factory new) doSomething: 2 with: 4'"
```

Comment:
again, maybe rename message name would be better than a comment. example can stay imo